### DISCUSSION TITLE
Welcome to programmer-calculator Discussions!

### DISCUSSION THREAD

**Discussion by alt-romes** [alt-romes/programmer-calculator#34] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hexactinal** [comment]
My experience compiling on Pop!_OS 20.10:

1. Followed the instructions in the readme...assumed that I already had `ncurses`, but according to [this website](https://www.garron.me/en/linux/install-ncurses-library-ubuntu-debian.html), I should have done `sudo apt-get install libncurses5-dev libncursesw5-dev` first before compiling. Oops, my mistake!
2. The `make` command gave me [this error](https://pastebin.com/eXPqx6Zg) while compiling.
3. After running `make`, I tried to move it to /usr/local/bin, but it gave another error (as shown in the pastebin link).
4. Somehow, I got it to work by running `cp pcalc ~/bin`. After that, running `pcalc` (even after doing cd ~) worked flawlessly!

This was my first time contributing to a project on GitHub, so I hope my findings are useful. Maybe you could add them to readme instructions for Ubuntu/Mint/Pop!_OS. Thanks, alt-romes!

**Reply by alt-romes** [reply]
I've updated the README with attention to your findings !
It should be correct now, and I've added the hint for installing ncurses.

You're also welcome to submit pull requests and issues if you encounter anything else

Thank you :)
~romes

**Reply by DevManu-de** [reply]
Hello @hexactinal,
the error (its actually a warning) you got during compilation is normal.
> src/operators.c:133:42: warning: left shift of negative value [-Wshift-negative-value]

Thanks for contributing.
DevManu-de
